cl/scrapers/utils.py

```diff
@@ -446,3 +449,49 @@ def scraped_citation_object_is_valid(citation_object: dict) -> bool:
         logger.error("Parsed reporter '%s' does not exist", parsed_reporter)
 
     return False
+
+
+def save_response(site: AbstractSite) -> None:
+    """Stores scrapers responses content and headers in a S3 bucket
+
+    This is passed to juriscraper's Site objects as the
+    `save_response_fn` argument, which will make Juriscraper
+    save every response
+
+    :param site: the Site object, used to access the saved response
+    :return None
+    """
+
+    storage = S3GlacierInstantRetrievalStorage()
+    response = site.request["response"]
+    headers = dict(response.headers)
+
+    scraper_id = site.court_id.split(".")[-1]
+    scrape_type = site.court_id.split(".")[1]  # opinions or oral args
+    now_str = datetime.now().strftime("%Y_%m/%d/%H_%M_%S")
```

Comment:
I think I'd suggest: 

```suggestion
    now_str = datetime.now().strftime("%Y/%m/%d/%H_%M_%S")
```

That'd give you daily separation, which seems generally good.